docs/fsharp/language-reference/slices.md

```diff
@@ -136,6 +136,17 @@ printSpan sp.[0..3] // [|1; 2; 3|]
 printSpan sp.[1..2] // |2; 3|]
 ```
 
+## Built-in F# slices are end-inclusive
+
+All intrinsic slices in F# are end-inclusive; that is, the upper bound is included in the slice. For a given slice with starting index `x` and ending index `y`, the resulting slice will include the `yth` value.
```

Comment:
```suggestion
All intrinsic slices in F# are end-inclusive; that is, the upper bound is included in the slice. For a given slice with starting index `x` and ending index `y`, the resulting slice will include the *yth* value.
```
"th" seems English-specific, so it should probably be localized (so I took it out of backticks).